model-optimizer/unit_tests/mo/utils/versions_checker_test.py

```diff
@@ -86,6 +88,41 @@ def test_append_version_list(self):
         for i, v in enumerate(req_list):
             self.assertEqual(v, ref_list[i])
 
+    def test_append_version_list_sys_neg(self):
+        v1 = "mxnet>=1.7.0 ; sys_platform != 'win32'"
+        req_list = list()
+        parse_and_filter_versions_list(v1, req_list, {'sys_platform': sys.platform})
+        ref_list = [('mxnet', '>=', '1.7.0')] if sys.platform != 'win32' else []
+        for i, v in enumerate(req_list):
+            self.assertEqual(v, ref_list[i])
+
+    def test_append_version_list_sys(self):
+        v1 = "mxnet>=1.7.0 ; sys_platform == 'linux'"
+        req_list = list()
+        platform = sys.platform
```

Comment:
Please assign the concrete value for `sys_platform` and test different cases. The behavior of tests must not be dependent on platform.

Reply:
Done, tests were updated.